Iterable-iOS-SDK/Iterable-iOS-SDK/IterableAPI.m

```diff
@@ -361,4 +368,21 @@ - (void)trackPurchase:(NSNumber *)total items:(NSArray<CommerceItem> *)items dat
     }
 }
 
-@end
+- (NSString*)pushServicePlatformToString:(PushServicePlatform)pushServicePlatform{
+    NSString *result = nil;
+    
+    switch(pushServicePlatform) {
+        case APNS:
+            result = @"APNS";
+            break;
+        case APNS_SANDBOX:
+            result = @"APNS_SANDBOX";
+            break;
+        default:
+            NSLog(@"Unexpected PushServicePlatform: %d", pushServicePlatform);
```

Comment:
try running in this case. what happens? result is gonna be nil, will it still serialize the dictionary properly in registerToken? just wanna make sure it wont except down the line